examples/Atlas/+atlasParams/Base.m

```diff
@@ -3,11 +3,12 @@
   % are fixed at compile time for a particular type of plan (walking, manipulating,
   % standing, recovery, etc.) but which can be switched out for a different set when
   % the plan type changes. 
-  properties(GetAccess=public, SetAccess=protected)
+  properties
     body_motion
```

Comment:
Done. Full documentation is in the BodyMotionData class, and I've added a note here pointing to that documentation.